### DISCUSSION TITLE
problem exporting collection

### DISCUSSION THREAD

**Discussion by RaoulL1** [retorquere/zotero-better-bibtex#2606] [General]
Hi, 

I was using better bibtex since many times with one collection. Since today, I cannot export anymore the collection using better bibtex or better biblatex  (it works with others format)

Debug log ID: M7QK6VBX-refs-euc

Thanks for your incredible work with this plug-in!!

**Comment by retorquere** [comment]
You're running out of memory. Disabling the cache will help some. Zotero 7 will be out soon and will resolve this -- Zotero 6 is 32bit on Windows and is memory constrained. Zotero 7 for Windows is 64 bit. BBT ccompat with Zotero 7 is tracked on #2522.

**Comment by RaoulL1** [comment]
Super now it works. Thank you very much. However now citation pick up on Visual Studio code doesn't work anymore (with any extension)

**Reply by RaoulL1** [reply]
the pick up window open, I can select the citation, it seems to work, but then when I press enter, the key is not printed in the document.

**Reply by retorquere** [reply]
Please upgrade to the latest version. If that still exhibits the problem, open a new issue.

**Reply by RaoulL1** [reply]
Yes, it works! Super. Thank you very much
